planning/behavior_path_planner/src/utils/start_planner/geometric_pull_out.cpp

```diff
@@ -80,15 +80,30 @@ boost::optional<PullOutPath> GeometricPullOut::plan(Pose start_pose, Pose goal_p
         parameters_.collision_check_margin)) {
     return {};
   }
+  const double velocity = parallel_parking_parameters_.forward_parking_velocity;
 
   if (parameters_.divide_pull_out_path) {
     output.partial_paths = planner_.getPaths();
     // insert stop velocity to first arc path end
+    const double arc_length_on_path =
+      motion_utils::calcArcLength(output.partial_paths.front().points);
     output.partial_paths.front().points.back().point.longitudinal_velocity_mps = 0.0;
+    output.pairs_terminal_velocity_and_accel.push_back(
+      std::make_pair(velocity, velocity * velocity / (2 * arc_length_on_path)));
```

Comment:
partial_paths size is two, but only one `pairs_terminal_velocity_and_accel` is ok?